src/mlj_interface.jl

```diff
@@ -94,15 +94,20 @@ function MMI.fit(m::KMeans, verbosity::Int, X)
 
     # fit model and get results
     verbose = verbosity > 0  # Display fitting operations if verbosity > 0
-    fitresult = ParallelKMeans.kmeans(algo, DMatrix, m.k;
+    result = ParallelKMeans.kmeans(algo, DMatrix, m.k;
                                       n_threads = m.threads, k_init=m.k_init,
                                       max_iters=m.max_iters, tol=m.tol, init=m.init,
                                       verbose=verbose)
 
+    cluster_labels = MMI.categorical(1:m.k)
+    fitresult = (result.centers, cluster_labels, result.converged)
     cache = nothing
-    report = (cluster_centers=fitresult.centers, iterations=fitresult.iterations,
-              converged=fitresult.converged, totalcost=fitresult.totalcost,
-              labels=fitresult.assignments)
+
+    report = (cluster_centers=result.centers, iterations=result.iterations,
+              converged=result.converged, totalcost=result.totalcost,
+              assignments=result.assignments, labels=cluster_labels)
+
+
     """
     # TODO: warn users about non convergence
```

Comment:
Shouldn't it be removed? It looks like next line do necessary warning.

Reply:
Yes, you are right. I will remove it now.